src/hardware/vga_draw.cpp

```diff
@@ -1635,24 +1572,9 @@ void VGA_SetupDrawing(uint32_t /*val*/)
 	case M_TEXT:
 		vga.draw.blocks=width;
 		doublewidth=(vga.seq.clocking_mode & 0x8) > 0;
-		if ((IS_VGA_ARCH) && (svgaCard==SVGA_None)) {
-			// vgaonly: allow 9-pixel wide fonts
-			if (vga.seq.clocking_mode&0x01) {
-				vga.draw.char9dot = false;
-				width*=8;
-			} else {
-				vga.draw.char9dot = true;
-				width*=9;
-				aspect_ratio *= 1.125;
-			}
-			VGA_DrawLine=VGA_TEXT_Xlat16_Draw_Line;
-			bpp = 16;
-		} else {
-			// not vgaonly: force 8-pixel wide fonts
-			width*=8; // 8 bit wide text font
-			vga.draw.char9dot = false;
-			VGA_DrawLine=VGA_TEXT_Draw_Line;
-		}
```

Comment:
Oh noes! I was hoping you'll make 9-pixel wide fonts the standard in all VGA modes but it seems you're nuking them? 😢 Anyway, that's something we can add back later I guess.

Reply:
Thanks @johnnovak - fixed.

### `machine = ega`

> EGA: 8x14 is the standard font, so 80x25 column text is displayed at 640x350 resolution

```
DISPLAY: Text 640x350 16 color (mode 03h) at 59.702 Hz VFR,
         scaled to 2132x1599 with 1.371 pixel aspect ratio
```

![2023-04-17_16-59](https://user-images.githubusercontent.com/1557255/232635548-b8d34d92-7263-4258-9d09-4ff3553403b1.png)


### `machine = svga_s3` (and other SVGA machines)

> VGA: most (all?) VGA adapters upgraded the font to 9x16, so 80x25 column text is displayed at 720x400 resolution

```
DISPLAY: Text 720x400 16 color (mode 03h) at 70.087 Hz VFR,
         scaled to 2132x1599 with 1.35 pixel aspect ratio
```

![2023-04-17_16-58](https://user-images.githubusercontent.com/1557255/232635432-adabad9a-ebf7-4abb-8a55-3a04730bf1d3.png)

![2023-04-17_16-57](https://user-images.githubusercontent.com/1557255/232635436-7db9d507-aa2a-4af2-9492-2ca173ffab37.png)